src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -145,6 +224,10 @@ class OwlViTObjectDetectionOutput(ModelOutput):
         vision_model_last_hidden_state (`torch.FloatTensor` of shape `(batch_size, num_patches + 1, hidden_size)`)):
             Last hidden states extracted from the [`OwlViTVisionModel`]. OWL-ViT represents images as a set of image
             patches where the total number of patches is (image_size / patch_size)**2.
+        text_model_output (Tuple[`BaseModelOutputWithPooling`]):
```

Comment:
This is another breaking change. Also, it's weird to have both a "vision_model_output_last_hidden_state" and "vision_model_output" in the outputs of the model. The last hidden states are already included in the latter.

CLIP for instance only includes "vision_model_output".

Reply:
I don't think it's a breaking charge as I haven't removed the last_hidden_states. I agree that it's weird to include both but removing them would be the breaking change.